packages/webapp/components/invite/Referral.tsx

```diff
@@ -0,0 +1,221 @@
+    // we don't need to send the params here explicitly
+    trackEvent({ event_name: AnalyticsEvent.AcceptGeneralReferralInvitation });
+
+    return handleAccept();
+  };
+
+  useEffect(() => {
+    if (genericReferral === GenericReferral.Control) {
+      return;
+    }
+
+    router.push(redirectTo);
+    // router is an unstable dependency
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [redirectTo, genericReferral]);
+
+  return (
+    <div
+      style={{
+        background: `url(${cloudinary.referralCampaign.genericReferral.backgroundDark})`,
+        backgroundSize: 'cover',
+      }}
+      className="flex overflow-hidden relative flex-col laptop:flex-row flex-auto items-center p-2 laptop:p-6 h-[100vh]"
+    >
+      <span className="absolute top-8 left-1/2 laptop:left-12 -translate-x-1/2 laptop:translate-x-0 scale-[1.375]">
+        <Logo showGreeting={false} position={LogoPosition.Relative} />
+      </span>
+      <div className="flex z-1 flex-col p-4 laptop:p-0 laptop:mt-0 laptop:ml-3 w-full laptopL:ml-[9.315rem] laptopL:max-w-[37.2rem] mt-[5rem]">
+        <span className="flex flex-col laptop:flex-row gap-3 laptopL:gap-6 items-center laptop:items-start mb-6 tablet:mb-8">
+          <ProfileImageLink
+            user={referringUser}
+            picture={{ size: isLaptopL ? 'xxxlarge' : 'xlarge' }}
+          />
+          <p className="my-auto text-center laptop:text-left typo-headline laptop:typo-title3 laptopL:typo-title2">
+            <span className="block laptop:inline">{referringUser.name}</span>
+            <span className="font-normal"> invited you to daily.dev</span>
+          </p>
+        </span>
+        <h1 className="w-full text-center laptop:text-left break-words-overflow typo-large-title tablet:typo-mega2 laptopL:typo-giga2">
+          The Homepage
+          <span className="block font-bold text-theme-color-cabbage">
+            {' '}
+            Developers Deserve
+          </span>
+        </h1>
+        <p className="mx-auto laptop:mx-0 mt-7 tablet:mt-8 max-w-sm laptopL:max-w-xl text-center laptop:text-left typo-title3 text-theme-label-tertiary">
+          Get a feed of the best developer news out there! Read more quality
+          articles.
+          <span className="tablet:block laptopL:inline"> Stay up to date.</span>
+        </p>
+        <Button
+          buttonSize={isMobile ? ButtonSize.Large : ButtonSize.XLarge}
+          className="p-4 mx-auto laptop:mx-0 mt-6 tablet:mt-12 max-w-[17.5rem] mobileL:max-w-[24.25rem] btn-primary"
+          tag="a"
+          href={downloadBrowserExtension}
+          onClick={() => {
+            handleAcceptClick();
+          }}
+          target="_blank"
+          loading={isLoading}
+          disabled={isLoading || isSuccess}
+        >
+          <FlexCentered className="gap-2 mobileL:typo-title3">
+            <BrowsersIcon
+              width={isMobile ? '58px' : '80px'}
+              height={isMobile ? '30px' : '40px'}
```

Comment:
We don't use pixels, and icons have sizes, it should reflect a icon size.

Reply:
The sizes of the icon component we have don't match up with this icon, probably because this button is in the landing pages we don't have this in the proper sizes here. 
Since we're adding overrides for the button to be a different size on what we have on the guidelines because we want to make it similar to the one in the landing page, maybe we are ok to keep these in pixels until it is decided this button will be properly introduced here? 🤔

Reply:
We must at least convert the values to `rem`. Seems like a one-off too, so I am cool if we will make it hard-coded. Though we should probably have it defined in the `className`.